link.go

```diff
@@ -53,21 +53,47 @@ unlinked QmVGtdTZdTFaLsaj2RwdVG8jcjNNcp1DE914DKZ2kHmXHw /home/user/go/src/github
 			Name:  "a,all",
 			Usage: "Remove all existing symlinks and reinstate the gx packages. Use with -r.",
 		},
+		cli.StringFlag{
+			Name:  "p,parent",
+			Usage: "Specify the path of the parent package of the linked dependency (default: CWD).",
+		},
+		cli.BoolFlag{
+			Name:  "s,sync",
+			Usage: "Synchronize the dependencies of the linked package with the ones of its parent.",
+		},
 	},
 	Action: func(c *cli.Context) error {
 		remove := c.Bool("remove")
 		all := c.Bool("all")
+		parentPackagePath := c.String("parent")
+		sync := c.Bool("sync")
+
+		if parentPackagePath == "" {
+			var err error
+			parentPackagePath, err = os.Getwd()
```

Comment:
Should probably use `gxutil.GetPackageRoot()`

Reply:
(Fixed in the new PR.)